.eslintrc.js

```diff
@@ -56,7 +56,8 @@ const config = deepMerge(defaultConfig, {
         '@typescript-eslint/require-await': 0,
         '@typescript-eslint/no-unsafe-return': 0,
         '@typescript-eslint/no-unsafe-call': 0,
-        '@typescript-eslint/no-empty-interface': 0
+        '@typescript-eslint/no-empty-interface': 0,
+        '@typescript-eslint/restrict-plus-operands': 0
```

Comment:
OK for now, but we shouldn't add too many of these exceptions. I just added these to make sure we didn't have to fix too many warnings at one time. The idea was to gradually remove these rules.